.github/linters/mlc_config.json

```diff
@@ -23,6 +23,12 @@
     },
     {
       "pattern": "^https://aai.egi.eu/proxy*"
+    },
+    {
+      "pattern": "^https://voms2.hellasgrid.gr:8443/voms*"
```

Comment:
The reason for these is to not flag the docs as having broken links, even when the services pointed to by these URLs are down?

Reply:
No, it's because it's a service with access restricted with client authentication and it gives a false negative.
I currently don't have a better solution than disabling the check of this URL.